cp3pt0-deployment/common/migrate_singleton.sh

```diff
@@ -168,6 +279,9 @@ function print_usage() {
     echo "   --yq string                                    File path to yq CLI. Default uses yq in your PATH"
     echo "   --operator-namespace string                    Required. Namespace to migrate Foundational services operator"
     echo "   --enable-licensing                             Set this flag to migrate ibm-licensing-operator"
+    echo "   --enable-license-service-reporter              Set this flag to install ibm-license-service-reporter-operator"
```

Comment:
Imrpove the description to add human name of Reporter

Reply:
improved